test/listitem.spec.js

```diff
@@ -0,0 +1,47 @@
+import { mount } from '@vue/test-utils';
+import listitem from '../src/components/listitem.vue';
+
+describe('listitem.vue', () => {
+  const wrapper = mount(listitem, {
+    propsData: {
+      entry: { text: "test a todo list" },
+    }
+  })
+  it('renders a list item', () =>{
+    expect(wrapper.html()).toContain('<div class="row mx-lg-n5">');
+    expect(wrapper.html()).toContain('<form class="text">');
+    expect(wrapper.html()).toContain('type="button" name="delete-button" id="buttonDelete"');
+    expect(wrapper.html()).toContain('type="button" name="edit-button" id="buttonEdit"');
+  });
+  it('doesn\'t show the edit input immediately', () => {
```

Comment:
I like the test case title. It's telling the reader that it will show the edit input in a later test case :+1: